casper/src/main/resources/RevVault.rho

```diff
@@ -83,6 +83,15 @@ in {
                 }
               }
             } |
+            // Used during genesis to create PoS vault with initial balance = sum of initial bonds.
+            for (@"createWithBalance", @ownerRevAddress, @amount, ret <- RevVault) {
```

Comment:
I don't understand the grammar here `ret <- RevVault`. Is that a way to make this method can only be called when it is in genesis block?

Reply:
That's what I was going for. It's a linear receive so it will only be used the first time it's called. `ret` is the return channel supplied by the corresponding produce in the PoS contract.